src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -957,8 +975,16 @@ public IActionResult ListWalletsFiles()
 
         /// <summary>
         /// Creates a new account for a wallet.
+        /// Accounts are given the name "account i", where i is an incremental index which starts at 0.
```

Comment:
Is it worth mentioning that accounts are created deterministically?

Reply:
Probably yes, What exactly do you mean by this? That they will always be created the same on any machine...

Reply:
Have a couple of things to fix/add so I can embellish the account explanation too.

So, I noticed the following:

1. RequestModels.cs: TxFeeEstimateRequest - Outpoint is never used - comment currently unsuitable - can the Outpoint member be removed?
2. RequestModels.cs: TxFeeEstimateRequest - OpReturnAmount and OpReturnAmount moved to the superclass since my documentation - have documentation for them elsewhere so will add.
3. NodeController.cs: Shutdown() - [Route("shutdown-node")] should go back to the original [Route("shutdown-node")]

Reply:
Actually, TxFeeEstimateRequest.Outpoint is used.